### DISCUSSION TITLE
Remember Me

### DISCUSSION THREAD

**Discussion by marioekaputta** [bahasa-ai/otomatis-guidance#22] [General]
Every time I test my API, I have to log in to the dev account over and over again. In the maker platform, we already have this connected account to be remembered, I think we should have one on the dev platform as well. Or did we already have this but I misconfigure?
![image](https://user-images.githubusercontent.com/71392755/129322038-d99bb9d6-5220-46d8-a13c-2d3e09630fef.png)

**Comment by jundialwan** [comment]
Well noted, Mario.

@mgilangjanuar needs to save the testing bundle in the Dev Platform. Do we already have the necessary field in DB for this?

This will make Dev easier when testing out multiple cases after finish creates trigger/action. Re-login for every case is very tedious.

**Comment by mgilangjanuar** [comment]
We'll figure out about this, maybe we can store them in bundle data with testing flag
